stories/Molecules/Navbar/Breadcrumb/breadcrumb-link.scss

```diff
@@ -0,0 +1,81 @@
+@import '../../../assets/scss/breakpoints';
+@import '../../../assets/scss/variables';
+@import '../../../assets/scss/mixins';
+
+.breadcrumb__wrapper {
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  font-size: 12px;
+  padding: 0;
+
+  li {
+    display: inline-block;
+    margin: 10px;
+    position: relative;
+
+    &::after {
+      @include translate(0,-50%);
+
+      background: url(#{$img-path}/Separator.svg) no-repeat left center;
+      content: '';
+      height: 10px;
+      position: absolute;
+      right: -11px;
+      top: 50%;
+      width: 4px;
+    }
+
+    span {
```

Comment:
@pallavikadam-srijan Even we can pass the class on span and then provide the styling on the class

Reply:
@surbhi90 I added a class on span <span className="last-item">